src/lib/fetchers/SWRConfigProvider.tsx

```diff
@@ -2,36 +2,21 @@ import { t } from "@transifex/native";
 import * as React from "react";
 import { toast } from "react-toastify";
 import { SWRConfig, type SWRConfiguration } from "swr";
-import { ErrorMessage } from "~/components/SWRError/ErrorMessage";
 import type ResourceError from "~/lib/fetchers/ResourceError";
 
 const globalSWRConfig: SWRConfiguration<unknown, ResourceError> = {
   onError: (error, key) => {
-    const toastId = key;
     if (error) {
       console.error("SWR error", error);
-      const errorElement = <ErrorMessage error={error} />;
-      toast.error(errorElement, {
-        toastId,
-        delay: 2000,
-        autoClose: false,
-        position: "bottom-right",
-        hideProgressBar: false,
-        closeOnClick: false,
-        pauseOnHover: true,
-        draggable: true,
-        progress: undefined,
-        theme: "light",
-      });
```

Comment:
This will disable the error messages for all SWR request. This is not what we want. In this case we want to exclude 404s from a very specific request and I think that should be handled in the fetcher itself.